### PR TITLE
Contact Form

### PR DESCRIPTION
**Pull request by mohamedSabry0**
Features implemented in this PR:
- Added form HTML skeleton.
- Implemented styling following the design pointed in the readme.
- Added form action.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -289,9 +289,18 @@ <h2>About me</h2>
...
-      <div class="contact">
-        <form action=""></form>
-      </div>
+      <section class="contact">
+        <h2>I Would love to here from you. If you have a project or a feature
+          that you would like to discuss, please get in touch giving some summary 
+          using this form.
+        </h2>
+        <form action="https://formspree.io/f/myylpdqq" method="POST">
+          <input type="text" name="full-name" placeholder="Name">
```
Comment (tahiry-dev) [inline index.html]: Please limit the number of characters name field could accept

File: index.html [hunk 2]
```diff
@@ -289,9 +289,18 @@ <h2>About me</h2>
...
+          <input type="email" name="email" placeholder="Email address" required>
+          <input type="text" maxlength="500" placeholder="message..." required>
+          <button type="submit" class="form-submit">Get in touch</button>
```
Comment (tahiry-dev) [inline index.html]: Please prevent your user to send the form if one or more fields are empty.
As a quick reminder, here is the screenshot of the requirements:
requirements

**Review by tahiry-dev — CHANGES_REQUESTED** [review]
Hello mohamedSabry0 👋
You did a great job creating this contact form 💪 💪  ,
 but we need to implement more control to our form before moving to the next step 🤞🤞

## Required Changes ♻️
_Kindly limit the number of characters your name field could accept._
_And add validation to your form._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by umairarshadbutt — APPROVED** [review]
Hi @mohamedSabry0 , 

Your project is complete!  You have implemented the suggested changes... I suggest you some optional change before merge :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Please remove `!important` from CSS file.
- Please update your live link.
- Please improve the contact form design.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by ahcodes — APPROVED** [review]

**Pull request merged**